### PR TITLE
Fix: updating star rating title

### PR DESCRIPTION
**Pull request by championshuttler**
Fixing #5042

### CODE REVIEW

File: src/ui/components/Rating/index.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ export class RatingBase extends React.Component {
         return i18n.sprintf(i18n.gettext('Rated %(rating)s out of 5.'),
           { rating: i18n.formatNumber(parseFloat(rating).toFixed(1)) });
       }
-      return i18n.gettext('This add-on has not been rated yet.');
+      return i18n.gettext('There are no ratings yet');
```
Comment (willdurand) [inline src/ui/components/Rating/index.js]: I think you should add a period at the end.

**Comment by championshuttler** [comment]
@willdurand is it looking fine now?

**Comment by willdurand** [comment]
@championshuttler yup, that's much better!

**Comment by pwalm** [comment]
Use "There are no ratings yet" when an add-on is ratings-less. Thanks for catching this!

File: src/ui/components/Rating/index.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ export class RatingBase extends React.Component {
         return i18n.sprintf(i18n.gettext('Rated %(rating)s out of 5.'),
           { rating: i18n.formatNumber(parseFloat(rating).toFixed(1)) });
       }
-      return i18n.gettext('This add-on has not been rated yet.');
+      return i18n.gettext('There are no ratings yet.');
```
Comment (willdurand) [inline src/ui/components/Rating/index.js]: Can you remove the period please? See: https://design.firefox.com/photon/copy/punctuation.html

If you could do that for the other texts in the `renderTitle()` method, that would be ❤️

**Comment by championshuttler** [comment]
@willdurand locally tests are not failing

**Comment by willdurand** [comment]
> @willdurand locally tests are not failing

tests don't fail actually, but I rebased your branch to make sure dependencies don't fail. Everything is fine with your PR, thanks a lot 👍

**Pull request merged**